packages/dhis-api/src/__tests__/DhisApi/testCodesToIds.js

```diff
@@ -25,10 +25,14 @@ export const testCodesToIds = () => {
     const ids = DATA_ELEMENTS.map(({ id }) => id);
     const fetchStub = jest.fn();
     when(fetchStub)
-      .calledWith('dataElements', {
-        fields: expect.arrayContaining(['id']),
-        filter: { comparator: 'in', code: expect.toBeOneOf(['[POP01,POP02]', '[POP02,POP01]']) },
-      })
+      .calledWith(
+        'dataElements',
+        {
+          fields: expect.arrayContaining(['id']),
+          filter: { comparator: 'in', code: expect.toBeOneOf(['[POP01,POP02]', '[POP02,POP01]']) },
+        },
+        undefined,
```

Comment:
The new version of `jest-when` seems to be stricter with expected args - you have to explicitly list `undefined`